examples/js/postprocessing/AdaptiveToneMappingPass.js

```diff
@@ -186,7 +186,7 @@ THREE.AdaptiveToneMappingPass.prototype = {
 			this.previousLuminanceRT.dispose();
 
 		}
-		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBFormat };
+		var pars = { minFilter: THREE.LinearFilter, magFilter: THREE.LinearFilter, format: THREE.RGBAFormat };
```

Comment:
Same as in #8415. Firefox does not render with `RGBFormat` and reports the well-known message:

_Error: WebGL: drawElements: Incomplete framebuffer: Status 0x8cd6: COLOR_ATTACHMENT0 has an effective format of RGB32F, which is not renderable_